When try to remove environment from apictl without logging out, it gives a NPE.

**Description:**
When a user trying to remove an environment without logging out from that environment it gives an error (Null Pointer Exception). Stack trace of the error is attached below.
```

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7f6df2]

goroutine 1 [running]:
github.com/wso2/product-apim-tooling/import-export-cli/utils.GetApiManagerEndpointOfEnv(...)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/utils/envManagementUtils.go:159
github.com/wso2/product-apim-tooling/import-export-cli/utils.GetInternalTokenEndpointOfEnv(0x7ffe3a1a6194, 0xa, 0xc00012c060, 0x29, 0x1c, 0x0)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/utils/envManagementUtils.go:347 +0x52
github.com/wso2/product-apim-tooling/import-export-cli/credentials.GetOAuthAccessToken(0xc000700520, 0x5, 0xc000700530, 0x5, 0xc00070b9a0, 0x1c, 0xc00070b9e0, 0x1c, 0x7ffe3a1a6194, 0xa, ...)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/credentials/credentials.go:73 +0x65
github.com/wso2/product-apim-tooling/import-export-cli/cmd.runLogout(0x7ffe3a1a6194, 0xa, 0xa, 0x1)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/cmd/logout.go:54 +0xd6
github.com/wso2/product-apim-tooling/import-export-cli/cmd.removeEnv(0x7ffe3a1a6194, 0xa, 0xc00012c060, 0x29, 0xc00012c030, 0x2a, 0x1, 0xc0004ce5b0)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/cmd/removeEnv.go:89 +0x297
github.com/wso2/product-apim-tooling/import-export-cli/cmd.executeRemoveEnvCmd(0x7ffe3a1a6194, 0xa, 0xc00012c060, 0x29, 0xc00012c030, 0x2a)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/cmd/removeEnv.go:55 +0x61
github.com/wso2/product-apim-tooling/import-export-cli/cmd.glob..func31(0x14d0440, 0xc0004ce5a0, 0x1, 0x1)
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/cmd/removeEnv.go:50 +0xd9
github.com/spf13/cobra.(*Command).execute(0x14d0440, 0xc0004ce560, 0x1, 0x1, 0x14d0440, 0xc0004ce560)
	/home/chamindu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x14d06a0, 0x44401a, 0x1472780, 0xc000000180)
	/home/chamindu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ea
github.com/spf13/cobra.(*Command).Execute(...)
	/home/chamindu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/wso2/product-apim-tooling/import-export-cli/cmd.Execute()
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/cmd/root.go:78 +0x31
main.main()
	/home/chamindu/Documents/Work/wso2/product-apim-tooling/import-export-cli/apictl.go:24 +0x20
```

**Suggested Labels:**
Affected/3.2.0, Type/Bug

**Suggested Assignees:**
Chamindu36

**Affected Product Version:**
3.2.0 

**OS, DB, other environment details and versions:**    
OS - Ubuntu 20.04 LTS
Java - JDK 1.8_252
APIM - 3.2.0 -Alpha

**Steps to reproduce:**

1. Create an environment using apictl.
2. Log in to that environment.
3. Perform some tasks as logged in user.
4. Try to remove the environment without logging out.


**Related Issues:**
<!-- Any related issues such as sub tasks, issues reported in other repositories (e.g component repositories), similar problems, etc. -->

--- a/import-export-cli/cmd/keys.go
+++ b/import-export-cli/cmd/keys.go
@@ -63,13 +63,6 @@ var genKeyCmd = &cobra.Command{
 //Subscribe the given API or API Product to the default application and generate an access token
 func getKeys() {
 
-	//Override the value of token endpoint if it is provided with get-keys command
-	if keyGenTokenEndpoint != "" {
-		fmt.Printf("New token endpoint '%s' is used as the token endpoint \n", keyGenTokenEndpoint)
-	} else {
-		var defaultTokenEndpoint = utils.GetTokenEndpointOfEnv(keyGenEnv,utils.MainConfigFilePath)
-		fmt.Printf("'%s' is used as the token endpoint \n", defaultTokenEndpoint)
-	}
 	cred, err := getCredentials(keyGenEnv)
 	if err != nil {
 		utils.HandleErrorAndExit("Error getting credentials", err)

--- a/import-export-cli/cmd/removeEnv.go
+++ b/import-export-cli/cmd/removeEnv.go
@@ -77,11 +77,6 @@ func removeEnv(envName, mainConfigFilePath, envKeysFilePath string) error {
 				return err
 			}
 		}
-		// remove env from mainConfig file (endpoints file)
-		err = utils.RemoveEnvFromMainConfigFile(envName, mainConfigFilePath)
-		if err != nil {
-			return err
-		}
 
 		// remove keys also if user has already logged into this environment
 		store, err := credentials.GetDefaultCredentialStore()
@@ -91,6 +86,13 @@ func removeEnv(envName, mainConfigFilePath, envKeysFilePath string) error {
 				return err
 			}
 		}
+
+		// remove env from mainConfig file (endpoints file)
+		err = utils.RemoveEnvFromMainConfigFile(envName, mainConfigFilePath)
+		if err != nil {
+			return err
+		}
+
 	} else {
 		// environment does not exist in mainConfig file (endpoints file). Nothing to remove
 		return errors.New("environment '" + envName + "' not found in " + mainConfigFilePath)

--- a/import-export-cli/integration/testUtils.go
+++ b/import-export-cli/integration/testUtils.go
@@ -252,9 +252,7 @@ func validateGetKeys(t *testing.T, args *apiGetKeyTestArgs) {
 
 		assert.Nil(t, err, "Error while getting key")
 
-		// Remove the string "'https://localhost:8243/token' is used as the token endpoint" from the result
-		stringWithToken := result[strings.Index(result, "\n")+1:]
-		invokeAPI(t, getResourceURL(args.apim, args.api), base.GetValueOfUniformResponse(stringWithToken), 200)
+		invokeAPI(t, getResourceURL(args.apim, args.api), base.GetValueOfUniformResponse(result), 200)
 		unsubscribeAPI(args.apim, args.ctlUser.username, args.ctlUser.password, args.api.ID)
 	}
 
@@ -266,9 +264,7 @@ func validateGetKeys(t *testing.T, args *apiGetKeyTestArgs) {
 
 		assert.Nil(t, err, "Error while getting key")
 
-		// Remove the string "'https://localhost:8243/token' is used as the token endpoint" from the result
-		stringWithToken := result[strings.Index(result, "\n")+1:]
-		invokeAPIProduct(t, getResourceURLForAPIProduct(args.apim, args.apiProduct), base.GetValueOfUniformResponse(stringWithToken), 200)
+		invokeAPIProduct(t, getResourceURLForAPIProduct(args.apim, args.apiProduct), base.GetValueOfUniformResponse(result), 200)
 		unsubscribeAPI(args.apim, args.ctlUser.username, args.ctlUser.password, args.apiProduct.ID)
 	}
 }

--- a/import-export-cli/shell-completions/apictl_zsh_completions.sh
+++ b/import-export-cli/shell-completions/apictl_zsh_completions.sh
@@ -17,13 +17,19 @@ case $state in
   ;;
   level2)
     case $words[2] in
-      add)
+      change-status)
         _arguments '2: :(api help)'
       ;;
-      change)
-        _arguments '2: :(help registry)'
+      export)
+        _arguments '2: :(api-product help)'
       ;;
-      change-status)
+      list)
+        _arguments '2: :(api-products apis apps envs help)'
+      ;;
+      remove)
+        _arguments '2: :(env help)'
+      ;;
+      add)
         _arguments '2: :(api help)'
       ;;
       delete)
@@ -38,17 +44,11 @@ case $state in
       uninstall)
         _arguments '2: :(api-operator help wso2am-operator)'
       ;;
-      export)
-        _arguments '2: :(api-product help)'
-      ;;
-      list)
-        _arguments '2: :(api-products apis apps envs help)'
-      ;;
-      remove)
-        _arguments '2: :(env help)'
-      ;;
       update)
         _arguments '2: :(api help)'
+      ;;
+      change)
+        _arguments '2: :(help registry)'
       ;;
       *)
         _arguments '*: :_files'

